justfile

```diff
@@ -84,3 +84,11 @@ check-features:
 # Fix feature propagation and formatting
 fix-features:
     zepter && zepter format features --fix
+
+# Test codec conformance
+test-codec-fixtures:
+    just test --features arbitrary --profile codec_conformance
```

Comment:
nit: Maybe we just call `conformance`? It feels we could implement outside of `codec`?